web/js/layoutManager.js

```diff
@@ -2,74 +2,61 @@
 LayoutViewport, ItemsHandler */
 
 !(function (global) {
-  'use strict';
+  let userLayout = null;
```

Comment:
We had not been using ES6 (e.g. `let`) because of support for IE. But OpenTok.js no longer supports IE (at least not in supported versions of OpenTok.js), so it's reasonable to do this here. But we should probably change all of the code to ES6 and lint it thoroughly in a separate PR.

Reply:
true. I am interested in such a task of migrating to ES6.

Now I will revert all such changes related to ES6